nvflare/private/fed/server/server_command_agent.py

```diff
@@ -43,34 +43,40 @@ def start(self, engine):
 
     def listen_command(self, engine):
```

Comment:
sever_command_agent.py  and command_agent.py in large part are duplicate, consider refactoring this ?

Reply:
The processing logic are similar, but not duplicate. I will try to extract the common part.

Reply:
done